themes-default/slim/views/vue-components/qualityChooser.mako

```diff
@@ -0,0 +1,199 @@
+    <div id="quality_chooser_wrapper">
+        <select v-model="selectedQualityPreset" name="quality_preset" class="form-control form-control-inline input-sm">
+            <option :value="0" :selected="0 === overallQuality" >Custom</option>
+            <option v-for="preset in qualityPresets" :value="preset" :selected="preset === overallQuality" :style="qualityPresetStrings[preset].endsWith('0p') ? 'padding-left: 15px;' : ''">{{qualityPresetStrings[preset]}}</option>
+        </select>
+        <div id="customQualityWrapper">
+            <div style="padding-left: 0;" v-if="customQuality">
+                <p><b><strong>Preferred</strong></b> qualities will replace those in <b><strong>allowed</strong></b>, even if they are lower.</p>
+                <div style="padding-right: 40px; text-align: left; float: left;">
+                    <h5>Allowed</h5>
+                    <select v-model="selectedAllowed" name="allowed_qualities" multiple="multiple" :size="allowedQualityList.length" class="form-control form-control-inline input-sm">
+                        <option v-for="quality in allowedQualityList" :selected="quality in allowedQualities" :value="quality">{{qualityStrings[quality]}}</option>
+                    </select>
+                </div>
+                <div style="text-align: left; float: left;">
+                    <h5>Preferred</h5>
+                    <select v-model="selectedPreffered" name="preferred_qualities" multiple="multiple" :size="preferredQualityList.length" class="form-control form-control-inline input-sm">
+                        <option v-for="quality in preferredQualityList" :selected="quality in preferredQualities" :value="quality">{{qualityStrings[quality]}}</option>
+                    </select>
+                </div>
+            </div>
+            <div id="qualityExplanation">
+                <h5><b>Quality setting explanation:</b></h5>
+                <h5 v-if="selectedPreffered.length === 0">This will download <b>any</b> of these qualities and then stops searching: <label id="allowedExplanation">{{allowedExplanation.join(', ')}}</label></h5>
+                <template v-else>
+                <h5>Downloads <b>any</b> of these qualities: <label id="allowedPreferredExplanation">{{allowedExplanation.join(', ')}}</label></h5>
+                <h5>But it will stop searching when one of these is downloaded:  <label id="preferredExplanation">{{preferredExplanation.join(', ')}}</label></h5>
+                </template>
+            </div>
+            <div v-f="seriesSlug">
+                <h5 class="red-text" id="backloggedEpisodes" v-html="backloggedEpisodes"></h5>
+            </div>
+            <div id="archive" v-if="archive">
+                <h5>
+                    <b>
+                        Archive downloaded episodes that are not currently in <a target="_blank" href="manage/backlogOverview/"><font color="blue"><u>backlog</u>.</font></a>
```

Comment:
This is not showing anything when changing to a lower quality. I suspect you wanted to subsittute the `.`